Code/Framework/AzToolsFramework/AzToolsFramework/UI/PropertyEditor/PropertyEnumComboBoxCtrl.hxx

```diff
@@ -58,7 +62,8 @@ namespace AzToolsFramework
         void onChildComboBoxValueChange(int comboBoxIndex);
 
     private:
-        QComboBox* m_pComboBox;
+        QComboBox* m_pComboBox = nullptr;
+        QToolButton* m_editButton = nullptr;
         AZStd::vector< AZStd::pair<AZ::s64, AZStd::string>  > m_enumValues;
```

Comment:
uber nit: Can we remove the white space around `> >` here (that's an old C++ thing when compilers used to think `>>` in this context was a shift operator 😅 Shows how old some of this code is! 😝)